SurfaceTopography/Container/Moments.py

```diff
@@ -0,0 +1,98 @@
+from ..Generic.Moments import (compute_iso_moment,
+                               compute_1d_moment, )
+import numpy as np
+from ..Container import SurfaceContainer
+
+
+def _bandwidth_count_from_profile(self, qx, unit, reliable=True):
+    """
+    Return number of topographies that include qx in their bandwidth.
```

Comment:
Please add a description of the individual parameters

Reply:
Also, is qx a length (if it is in bandwidth) or is qx a wavevector? In the latter case the description should be adjusted, in the former case I would give qx a different name.